tutorials/asr/ASR_with_NeMo.ipynb

```diff
@@ -0,0 +1,960 @@
+        "                transcript = transcript.replace('<s>', '').replace('</s>', '')\n",
+        "                transcript = transcript.strip()\n",
+        "\n",
+        "                file_id = line[line.find('(')+1 : -2]  # e.g. \"cen4-fash-b\"\n",
+        "                audio_path = os.path.join(\n",
+        "                    data_dir, wav_path,\n",
+        "                    file_id[file_id.find('-')+1 : file_id.rfind('-')],\n",
+        "                    file_id + '.wav')\n",
+        "\n",
+        "                duration = librosa.core.get_duration(filename=audio_path)\n",
+        "\n",
+        "                # Write the metadata to the manifest\n",
+        "                metadata = {\n",
+        "                    \"audio_filepath\": audio_path,\n",
+        "                    \"duration\": duration,\n",
+        "                    \"text\": transcript\n",
+        "                }\n",
+        "                json.dump(metadata, fout)\n",
+        "                fout.write('\\n')\n",
+        "                \n",
+        "# Building Manifests\n",
+        "print(\"******\")\n",
+        "train_transcripts = data_dir + '/an4/etc/an4_train.transcription'\n",
+        "train_manifest = data_dir + '/an4/train_manifest.json'\n",
+        "if not os.path.isfile(train_manifest):\n",
+        "    build_manifest(train_transcripts, train_manifest, 'an4/wav/an4_clstk')\n",
+        "    print(\"Training manifest created.\")\n",
+        "\n",
+        "test_transcripts = data_dir + '/an4/etc/an4_test.transcription'\n",
+        "test_manifest = data_dir + '/an4/test_manifest.json'\n",
+        "if not os.path.isfile(test_manifest):\n",
+        "    build_manifest(test_transcripts, test_manifest, 'an4/wav/an4test_clstk')\n",
+        "    print(\"Test manifest created.\")\n",
+        "print(\"***Done***\")"
+      ],
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "W2fShQzRzo-M",
+        "colab_type": "text"
+      },
+      "source": [
+        "### Specifying Our Model with a YAML Config File\n",
+        "\n",
+        "For this tutorial, we'll build a *Jasper_4x1 model*, with `K=4` blocks of single (`R=1`) sub-blocks and a *greedy CTC decoder*, using the configuration found in `./configs/config.yaml`.\n",
+        "\n",
+        "If we open up this config file, we find model section which describes architecture of our model. A model contains an entry labeled `encoder`, with a field called `jasper` that contains a list with multiple entries. Each of the members in this list specifies one block in our model, and looks something like this:\n",
+        "```\n",
+        "- filters: 256\n",
+        "  repeat: 1\n",
+        "  kernel: [11]\n",
+        "  stride: [2]\n",
+        "  dilation: [1]\n",
+        "  dropout: 0.2\n",
+        "  residual: false\n",
```

Comment:
I just realized that this is slightly outdated now. Could you update this to add the new fields? (separable, se, se_context_size).